### ISSUE TITLE
.rpm support

### ISSUE THREAD

**Issue by lauren-elise-thomas** [iabem97/komorebi#75]
For Fedora/RHEL/CentOS, OpenSUSE, etcetera

**Comment by elsieholmes** [comment]
Yeah, it'd be great. I had to manually compile it on my OpenSUSE Tumbleweed.

**Comment by tim77** [comment]
📦 [Packaged in COPR](https://copr.fedorainfracloud.org/coprs/atim/komorebi/) for Fedora, but work in progress...

**Comment by aRealCyborg** [comment]
I would love to have this for Opensuse.

**Comment by Larsenist** [comment]
> 📦 [Packaged in COPR](https://copr.fedorainfracloud.org/coprs/atim/komorebi/) for Fedora, but work in progress...

When trying to install, there's no repositories providing vt323-fonts
Looking on [Koji](https://koji.fedoraproject.org/koji/packageinfo?tagStart=0&packageID=19142&tagOrder=name&buildOrder=-completion_time&buildStart=0#taglist), it looks like it was last included in F30
